test/measure.service.test.js

```diff
@@ -0,0 +1,127 @@
+require('../src/util/dbconfig');
+const testMeasure = require('./fixtures/testMeasure.json');
+const testLibrary = require('./fixtures/testLibrary.json');
+const testPatient = require('./fixtures/testPatient.json');
+const { testSetup, cleanUpDb } = require('./populateTestData');
+const supertest = require('supertest');
+const { buildConfig } = require('../src/util/config');
+const { initialize } = require('../src/server/server');
+const config = buildConfig();
+const server = initialize(config);
+
+
+const postRequest = {
+  encoding: 'utf8',
+  url: 'http://localhost:3000',
+  json: true,
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json+fhir'
+  },
+  body: JSON.stringify(testMeasure)
+};
+const putRequest = {
+  encoding: 'utf8',
+  url: 'http://localhost:3000',
+  json: true,
+  method: 'PUT',
+  headers: {
+    'Content-Type': 'application/json+fhir'
+  },
+  body: JSON.stringify(testMeasure)
+};
+const getRequest = {
+  encoding: 'utf8',
+  url: 'http://localhost:3000',
+  json: true,
+  method: 'GET',
+  headers: {
+    'Content-Type': 'application/json+fhir'
+  },
+  body: JSON.stringify(testMeasure)
+}
+
+const deleteRequest = {
+  encoding: 'utf8',
+  url: 'http://localhost:3000',
+  json: true,
+  method: 'DELETE',
+  headers: {
+    'Content-Type': 'application/json+fhir'
+  },
+  body: JSON.stringify(testMeasure)
+}
+describe('measure.service', () => {
+  beforeAll(async () => {
+    await testSetup(testMeasure, testPatient, testLibrary);
+  });
+  afterAll(async () => {
+    await cleanUpDb();
+  });
+  test('test', async () => {
+    await supertest(server.app).get('/4_0_0/metadata').expect(200);
+  });
```

Comment:
```suggestion
  test('test', async () => {
    await supertest(server.app).get('/4_0_0/metadata').expect(200);
  });
```

This can be removed. It was just to make sure the supertest syntax was correct IIRC, but we don't need it anymore